dissect/target/plugins/os/windows/dpapi/dpapi.py

```diff
@@ -144,12 +144,36 @@ def _users(self) -> dict[str, str]:
         """Cached map of username to SID."""
         return {user.name: user.sid for user in self.target.users()}
 
-    def decrypt_system_blob(self, data: bytes) -> bytes:
-        """Decrypt the given bytes using the SYSTEM master key."""
-        return self.decrypt_user_blob(data, sid=self.SYSTEM_SID)
+    def decrypt_system_blob(self, data: bytes, **kwargs) -> bytes:
+        """Decrypt the given bytes using the SYSTEM master key.
 
-    def decrypt_user_blob(self, data: bytes, username: str | None = None, sid: str | None = None) -> bytes:
-        """Decrypt the given bytes using the master key of the given SID or username."""
+        Args:
+            data: Bytes of DPAPI system blob to decrypt.
+            **kwargs: Arbitrary named arguments to pass to ``DPAPIBlob.decrypt`` function.
+
+        Raises:
+            ValueError: When conditions to decrypt are not met or if decrypting failed.
+
+        Returns:
+            Decrypted bytes.
+        """
+        return self.decrypt_user_blob(data, sid=self.SYSTEM_SID, **kwargs)
+
+    def decrypt_user_blob(self, data: bytes, username: str | None = None, sid: str | None = None, **kwargs) -> bytes:
+        """Decrypt the given bytes using the master key of the given SID or username.
+
+        Args:
+            data: Bytes of DPAPI blob to decrypt.
+            username: Username of the owner of the DPAPI blob.
+            sid: SID of the owner of the DPAPI blob.
+            **kwargs: Arbitrary named arguments to pass to DPAPIBlob.decrypt function.
```

Comment:
```suggestion
            **kwargs: Arbitrary named arguments to pass to :meth:`DPAPIBlob.decrypt <dissect.target.plugins.os.windows.dpapi.blob.Blob.decrypt>` function.
```
I think that's the correct Sphinx syntax?